sunpy/net/base_client.py

```diff
@@ -130,6 +133,37 @@ def __init_subclass__(cls, *args, **kwargs):
             if name not in attrs.__all__:
                 attrs.__all__.append(name)
 
+        # Register client attrs after it has regsitered its own attrs
+        from sunpy.net import attr
+        values = cls.register_values()
+        # If the client has no support, we won't try to register attrs
+        if values:
+            attr.Attr.update_values({cls: cls.register_values()})
```

Comment:
Just a really silly optimisation:

```suggestion
            attr.Attr.update_values({cls: values})
```

Reply:
Fixed.